.github/workflows/validate-pull-request.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: olafurpg/setup-scala@v7
+      - uses: olafurpg/setup-scala@v10
```

Comment:
PR validation seems busted with this error:

```
Run olafurpg/setup-scala@v7
2
  with:
3
    java-version: graalvm@20.0.0
4
    jabba-version: 0.11.2
5
Installing Java version 'graalvm@20.0.0'
6
Error: Unable to process command '::set-env name=CI,::true' successfully.
```
This is me testing the newer version of this action to see if it fixes the issue.